### ISSUE TITLE
--py-solve doesn't actually generate the Python interface

### ISSUE THREAD

**Issue by scopatz** [ergs/transmutagen#6]
I am working on pyne integration (in the `pyne/src` dir) and running the command:

```sh
$ python -m transmutagen.gensolve --py-solve --namespace=pyne_cram --outfile=cram.c
```

and I am only seeing the cram.c and cram.h file generated...

**Comment by scopatz** [comment]
Oh, I guess that pysolve isn't generated at all...

**Comment by scopatz** [comment]
Which makes the gensolve docs misleading:

```
--py-solve            Generate code for py_solve.
```

**Comment by asmeurer** [comment]
It should probably be clearer. py_solve isn't a generated module. It's a hard-coded test module that lives alongside transmutagen. `python -m transmutagen.gensolve --py-solve` generates the generated code of py_solve (`py_solve.c` and `py_solve.h`). 

If you need generated Cython code that isn't implemented yet. For pyne, I would just copy py_solve.pyx and modify it for pyne's needs, then generate the appropriate C file with `transmutagen.gensolve`

**Comment by scopatz** [comment]
Yeah, I am doing something like that for pyne.  I think that this is effectively a docs issue at this point.

**Comment by asmeurer** [comment]
We don't have much docs yet (just the README, docstrings, and command line help). We should build some Sphinx docs at some point. How high priority do you think it is?

**Comment by scopatz** [comment]
I would put it at a moderate priority, only if we wanted to push a paper out through JOSS

**Comment by scopatz** [comment]
We should think about if we want to do that before or after the full paper

**Comment by asmeurer** [comment]
The only question is if having one published affects the publishing of the other (either positively or negatively). I thought I remembered Lorena saying that software that will have full blown papers shouldn't submit to JOSS (but I could be wrong). 

JOSS would be easier. We just need better documentation for it (unless there's something else they require that I'm forgetting about).

**Comment by scopatz** [comment]
I think that having a JOSS paper could slightly positively affect the full paper by having a DOI that we can cite to the software that isn't just a figshare or some such. However, I don't want to delay the full paper in any way. It seems like we can add a citation during review / edits if we want.

**Comment by scopatz** [comment]
In other word, we can do the JOSS paper later
